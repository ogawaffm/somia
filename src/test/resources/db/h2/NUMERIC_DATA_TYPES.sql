DROP TABLE IF EXISTS NUMERIC_DATA_TYPES;
CREATE TABLE NUMERIC_DATA_TYPES (

    TINYINT_COL TINYINT,
    SMALLINT_COL SMALLINT,
    INT_COL INT,
    INTEGER_COL INTEGER,
    BIGINT_COL BIGINT,

    REAL_COL REAL,

    FLOAT_COL FLOAT,
    FLOAT_1_COL FLOAT(1),
    FLOAT_24_COL FLOAT(24),
    FLOAT_25_COL FLOAT(25),
    FLOAT_53_COL FLOAT(53),

    DOUBLE_COL DOUBLE,
    DOUBLE_PRECISION_COL DOUBLE PRECISION,

    DECFLOAT_COL DECFLOAT,
    DECFLOAT_1_COL DECFLOAT(1),
    DECFLOAT_100000_COL DECFLOAT(100000),

    DEC_COL DEC,
    DECIMAL_COL DECIMAL,
    NUMERIC_COL NUMERIC,

    DEC_1_COL DEC(1),
    DECIMAL_1_COL DECIMAL(1),
    NUMERIC_1_COL NUMERIC(1),

    DEC_1_0_COL DEC(1, 0),
    DECIMAL_1_0_COL DECIMAL(1, 0),
    NUMERIC_1_0_COL NUMERIC(1, 0),

    DEC_1_1_COL DEC(1, 1),
    DECIMAL_1_1_COL DECIMAL(1, 1),
    NUMERIC_1_1_COL NUMERIC(1, 1),

    DEC_100000_COL DEC(100000),
    DECIMAL_100000_COL DECIMAL(100000),
    NUMERIC_100000_COL NUMERIC(100000),

    DEC_100000_0_COL DEC(100000, 0),
    DECIMAL_100000_0_COL DECIMAL(100000, 0),
    NUMERIC_100000_0_COL NUMERIC(100000, 0),

    DEC_100000_1_COL DEC(100000, 1),
    DECIMAL_100000_1_COL DECIMAL(100000, 1),
    NUMERIC_100000_1_COL NUMERIC(100000, 1)

)