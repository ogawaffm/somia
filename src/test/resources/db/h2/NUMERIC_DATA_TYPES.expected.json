{
  "name" : "NUMERIC_DATA_TYPES",
  "catalogName" : "TEST",
  "schemaName" : "TEST",
  "typeName" : "BASE TABLE",
  "comment" : "",
  "columns" : [ {
    "name" : "TINYINT_COL",
    "type" : {
      "name" : "TINYINT",
      "typeNumber" : -6,
      "className" : "java.lang.Byte",
      "precision" : 8,
      "scale" : 0,
      "isNullable" : true,
      "isAutoIncrement" : false,
      "numericPrecisionRadix" : 2,
      "isCaseSensitive" : false,
      "isSearchable" : true,
      "isSigned" : true,
      "isCurrency" : false,
      "columnDisplaySize" : 4,
      "baseType" : null
    },
    "comment" : "",
    "collation" : "",
    "defaultDefinition" : ""
  }, {
    "name" : "SMALLINT_COL",
    "type" : {
      "name" : "SMALLINT",
      "typeNumber" : 5,
      "className" : "java.lang.Short",
      "precision" : 16,
      "scale" : 0,
      "isNullable" : true,
      "isAutoIncrement" : false,
      "numericPrecisionRadix" : 2,
      "isCaseSensitive" : false,
      "isSearchable" : true,
      "isSigned" : true,
      "isCurrency" : false,
      "columnDisplaySize" : 6,
      "baseType" : null
    },
    "comment" : "",
    "collation" : "",
    "defaultDefinition" : ""
  }, {
    "name" : "INT_COL",
    "type" : {
      "name" : "INTEGER",
      "typeNumber" : 4,
      "className" : "java.lang.Integer",
      "precision" : 32,
      "scale" : 0,
      "isNullable" : true,
      "isAutoIncrement" : false,
      "numericPrecisionRadix" : 2,
      "isCaseSensitive" : false,
      "isSearchable" : true,
      "isSigned" : true,
      "isCurrency" : false,
      "columnDisplaySize" : 11,
      "baseType" : null
    },
    "comment" : "",
    "collation" : "",
    "defaultDefinition" : ""
  }, {
    "name" : "INTEGER_COL",
    "type" : {
      "name" : "INTEGER",
      "typeNumber" : 4,
      "className" : "java.lang.Integer",
      "precision" : 32,
      "scale" : 0,
      "isNullable" : true,
      "isAutoIncrement" : false,
      "numericPrecisionRadix" : 2,
      "isCaseSensitive" : false,
      "isSearchable" : true,
      "isSigned" : true,
      "isCurrency" : false,
      "columnDisplaySize" : 11,
      "baseType" : null
    },
    "comment" : "",
    "collation" : "",
    "defaultDefinition" : ""
  }, {
    "name" : "BIGINT_COL",
    "type" : {
      "name" : "BIGINT",
      "typeNumber" : -5,
      "className" : "java.lang.Long",
      "precision" : 64,
      "scale" : 0,
      "isNullable" : true,
      "isAutoIncrement" : false,
      "numericPrecisionRadix" : 2,
      "isCaseSensitive" : false,
      "isSearchable" : true,
      "isSigned" : true,
      "isCurrency" : false,
      "columnDisplaySize" : 20,
      "baseType" : null
    },
    "comment" : "",
    "collation" : "",
    "defaultDefinition" : ""
  }, {
    "name" : "REAL_COL",
    "type" : {
      "name" : "REAL",
      "typeNumber" : 7,
      "className" : "java.lang.Float",
      "precision" : 24,
      "scale" : 0,
      "isNullable" : true,
      "isAutoIncrement" : false,
      "numericPrecisionRadix" : 2,
      "isCaseSensitive" : false,
      "isSearchable" : true,
      "isSigned" : true,
      "isCurrency" : false,
      "columnDisplaySize" : 24,
      "baseType" : null
    },
    "comment" : "",
    "collation" : "",
    "defaultDefinition" : ""
  }, {
    "name" : "FLOAT_COL",
    "type" : {
      "name" : "DOUBLE PRECISION",
      "typeNumber" : 6,
      "className" : "java.lang.Double",
      "precision" : 53,
      "scale" : 0,
      "isNullable" : true,
      "isAutoIncrement" : false,
      "numericPrecisionRadix" : 2,
      "isCaseSensitive" : false,
      "isSearchable" : true,
      "isSigned" : true,
      "isCurrency" : false,
      "columnDisplaySize" : 24,
      "baseType" : null
    },
    "comment" : "",
    "collation" : "",
    "defaultDefinition" : ""
  }, {
    "name" : "FLOAT_1_COL",
    "type" : {
      "name" : "REAL",
      "typeNumber" : 6,
      "className" : "java.lang.Double",
      "precision" : 24,
      "scale" : 0,
      "isNullable" : true,
      "isAutoIncrement" : false,
      "numericPrecisionRadix" : 2,
      "isCaseSensitive" : false,
      "isSearchable" : true,
      "isSigned" : true,
      "isCurrency" : false,
      "columnDisplaySize" : 24,
      "baseType" : null
    },
    "comment" : "",
    "collation" : "",
    "defaultDefinition" : ""
  }, {
    "name" : "FLOAT_24_COL",
    "type" : {
      "name" : "REAL",
      "typeNumber" : 6,
      "className" : "java.lang.Double",
      "precision" : 24,
      "scale" : 0,
      "isNullable" : true,
      "isAutoIncrement" : false,
      "numericPrecisionRadix" : 2,
      "isCaseSensitive" : false,
      "isSearchable" : true,
      "isSigned" : true,
      "isCurrency" : false,
      "columnDisplaySize" : 24,
      "baseType" : null
    },
    "comment" : "",
    "collation" : "",
    "defaultDefinition" : ""
  }, {
    "name" : "FLOAT_25_COL",
    "type" : {
      "name" : "DOUBLE PRECISION",
      "typeNumber" : 6,
      "className" : "java.lang.Double",
      "precision" : 53,
      "scale" : 0,
      "isNullable" : true,
      "isAutoIncrement" : false,
      "numericPrecisionRadix" : 2,
      "isCaseSensitive" : false,
      "isSearchable" : true,
      "isSigned" : true,
      "isCurrency" : false,
      "columnDisplaySize" : 24,
      "baseType" : null
    },
    "comment" : "",
    "collation" : "",
    "defaultDefinition" : ""
  }, {
    "name" : "FLOAT_53_COL",
    "type" : {
      "name" : "DOUBLE PRECISION",
      "typeNumber" : 6,
      "className" : "java.lang.Double",
      "precision" : 53,
      "scale" : 0,
      "isNullable" : true,
      "isAutoIncrement" : false,
      "numericPrecisionRadix" : 2,
      "isCaseSensitive" : false,
      "isSearchable" : true,
      "isSigned" : true,
      "isCurrency" : false,
      "columnDisplaySize" : 24,
      "baseType" : null
    },
    "comment" : "",
    "collation" : "",
    "defaultDefinition" : ""
  }, {
    "name" : "DOUBLE_COL",
    "type" : {
      "name" : "DOUBLE PRECISION",
      "typeNumber" : 8,
      "className" : "java.lang.Double",
      "precision" : 53,
      "scale" : 0,
      "isNullable" : true,
      "isAutoIncrement" : false,
      "numericPrecisionRadix" : 2,
      "isCaseSensitive" : false,
      "isSearchable" : true,
      "isSigned" : true,
      "isCurrency" : false,
      "columnDisplaySize" : 24,
      "baseType" : null
    },
    "comment" : "",
    "collation" : "",
    "defaultDefinition" : ""
  }, {
    "name" : "DOUBLE_PRECISION_COL",
    "type" : {
      "name" : "DOUBLE PRECISION",
      "typeNumber" : 8,
      "className" : "java.lang.Double",
      "precision" : 53,
      "scale" : 0,
      "isNullable" : true,
      "isAutoIncrement" : false,
      "numericPrecisionRadix" : 2,
      "isCaseSensitive" : false,
      "isSearchable" : true,
      "isSigned" : true,
      "isCurrency" : false,
      "columnDisplaySize" : 24,
      "baseType" : null
    },
    "comment" : "",
    "collation" : "",
    "defaultDefinition" : ""
  }, {
    "name" : "DECFLOAT_COL",
    "type" : {
      "name" : "DECFLOAT",
      "typeNumber" : 2,
      "className" : "java.math.BigDecimal",
      "precision" : 100000,
      "scale" : 0,
      "isNullable" : true,
      "isAutoIncrement" : false,
      "numericPrecisionRadix" : 10,
      "isCaseSensitive" : false,
      "isSearchable" : true,
      "isSigned" : true,
      "isCurrency" : false,
      "columnDisplaySize" : 100000,
      "baseType" : null
    },
    "comment" : "",
    "collation" : "",
    "defaultDefinition" : ""
  }, {
    "name" : "DECFLOAT_1_COL",
    "type" : {
      "name" : "DECFLOAT",
      "typeNumber" : 2,
      "className" : "java.math.BigDecimal",
      "precision" : 1,
      "scale" : 0,
      "isNullable" : true,
      "isAutoIncrement" : false,
      "numericPrecisionRadix" : 10,
      "isCaseSensitive" : false,
      "isSearchable" : true,
      "isSigned" : true,
      "isCurrency" : false,
      "columnDisplaySize" : 1,
      "baseType" : null
    },
    "comment" : "",
    "collation" : "",
    "defaultDefinition" : ""
  }, {
    "name" : "DECFLOAT_100000_COL",
    "type" : {
      "name" : "DECFLOAT",
      "typeNumber" : 2,
      "className" : "java.math.BigDecimal",
      "precision" : 100000,
      "scale" : 0,
      "isNullable" : true,
      "isAutoIncrement" : false,
      "numericPrecisionRadix" : 10,
      "isCaseSensitive" : false,
      "isSearchable" : true,
      "isSigned" : true,
      "isCurrency" : false,
      "columnDisplaySize" : 100000,
      "baseType" : null
    },
    "comment" : "",
    "collation" : "",
    "defaultDefinition" : ""
  }, {
    "name" : "DEC_COL",
    "type" : {
      "name" : "DECIMAL",
      "typeNumber" : 3,
      "className" : "java.math.BigDecimal",
      "precision" : 100000,
      "scale" : 0,
      "isNullable" : true,
      "isAutoIncrement" : false,
      "numericPrecisionRadix" : 10,
      "isCaseSensitive" : false,
      "isSearchable" : true,
      "isSigned" : true,
      "isCurrency" : false,
      "columnDisplaySize" : 100000,
      "baseType" : null
    },
    "comment" : "",
    "collation" : "",
    "defaultDefinition" : ""
  }, {
    "name" : "DECIMAL_COL",
    "type" : {
      "name" : "DECIMAL",
      "typeNumber" : 3,
      "className" : "java.math.BigDecimal",
      "precision" : 100000,
      "scale" : 0,
      "isNullable" : true,
      "isAutoIncrement" : false,
      "numericPrecisionRadix" : 10,
      "isCaseSensitive" : false,
      "isSearchable" : true,
      "isSigned" : true,
      "isCurrency" : false,
      "columnDisplaySize" : 100000,
      "baseType" : null
    },
    "comment" : "",
    "collation" : "",
    "defaultDefinition" : ""
  }, {
    "name" : "NUMERIC_COL",
    "type" : {
      "name" : "NUMERIC",
      "typeNumber" : 2,
      "className" : "java.math.BigDecimal",
      "precision" : 100000,
      "scale" : 0,
      "isNullable" : true,
      "isAutoIncrement" : false,
      "numericPrecisionRadix" : 10,
      "isCaseSensitive" : false,
      "isSearchable" : true,
      "isSigned" : true,
      "isCurrency" : false,
      "columnDisplaySize" : 100000,
      "baseType" : null
    },
    "comment" : "",
    "collation" : "",
    "defaultDefinition" : ""
  }, {
    "name" : "DEC_1_COL",
    "type" : {
      "name" : "DECIMAL",
      "typeNumber" : 3,
      "className" : "java.math.BigDecimal",
      "precision" : 1,
      "scale" : 0,
      "isNullable" : true,
      "isAutoIncrement" : false,
      "numericPrecisionRadix" : 10,
      "isCaseSensitive" : false,
      "isSearchable" : true,
      "isSigned" : true,
      "isCurrency" : false,
      "columnDisplaySize" : 1,
      "baseType" : null
    },
    "comment" : "",
    "collation" : "",
    "defaultDefinition" : ""
  }, {
    "name" : "DECIMAL_1_COL",
    "type" : {
      "name" : "DECIMAL",
      "typeNumber" : 3,
      "className" : "java.math.BigDecimal",
      "precision" : 1,
      "scale" : 0,
      "isNullable" : true,
      "isAutoIncrement" : false,
      "numericPrecisionRadix" : 10,
      "isCaseSensitive" : false,
      "isSearchable" : true,
      "isSigned" : true,
      "isCurrency" : false,
      "columnDisplaySize" : 1,
      "baseType" : null
    },
    "comment" : "",
    "collation" : "",
    "defaultDefinition" : ""
  }, {
    "name" : "NUMERIC_1_COL",
    "type" : {
      "name" : "NUMERIC",
      "typeNumber" : 2,
      "className" : "java.math.BigDecimal",
      "precision" : 1,
      "scale" : 0,
      "isNullable" : true,
      "isAutoIncrement" : false,
      "numericPrecisionRadix" : 10,
      "isCaseSensitive" : false,
      "isSearchable" : true,
      "isSigned" : true,
      "isCurrency" : false,
      "columnDisplaySize" : 1,
      "baseType" : null
    },
    "comment" : "",
    "collation" : "",
    "defaultDefinition" : ""
  }, {
    "name" : "DEC_1_0_COL",
    "type" : {
      "name" : "DECIMAL",
      "typeNumber" : 3,
      "className" : "java.math.BigDecimal",
      "precision" : 1,
      "scale" : 0,
      "isNullable" : true,
      "isAutoIncrement" : false,
      "numericPrecisionRadix" : 10,
      "isCaseSensitive" : false,
      "isSearchable" : true,
      "isSigned" : true,
      "isCurrency" : false,
      "columnDisplaySize" : 1,
      "baseType" : null
    },
    "comment" : "",
    "collation" : "",
    "defaultDefinition" : ""
  }, {
    "name" : "DECIMAL_1_0_COL",
    "type" : {
      "name" : "DECIMAL",
      "typeNumber" : 3,
      "className" : "java.math.BigDecimal",
      "precision" : 1,
      "scale" : 0,
      "isNullable" : true,
      "isAutoIncrement" : false,
      "numericPrecisionRadix" : 10,
      "isCaseSensitive" : false,
      "isSearchable" : true,
      "isSigned" : true,
      "isCurrency" : false,
      "columnDisplaySize" : 1,
      "baseType" : null
    },
    "comment" : "",
    "collation" : "",
    "defaultDefinition" : ""
  }, {
    "name" : "NUMERIC_1_0_COL",
    "type" : {
      "name" : "NUMERIC",
      "typeNumber" : 2,
      "className" : "java.math.BigDecimal",
      "precision" : 1,
      "scale" : 0,
      "isNullable" : true,
      "isAutoIncrement" : false,
      "numericPrecisionRadix" : 10,
      "isCaseSensitive" : false,
      "isSearchable" : true,
      "isSigned" : true,
      "isCurrency" : false,
      "columnDisplaySize" : 1,
      "baseType" : null
    },
    "comment" : "",
    "collation" : "",
    "defaultDefinition" : ""
  }, {
    "name" : "DEC_1_1_COL",
    "type" : {
      "name" : "DECIMAL",
      "typeNumber" : 3,
      "className" : "java.math.BigDecimal",
      "precision" : 1,
      "scale" : 1,
      "isNullable" : true,
      "isAutoIncrement" : false,
      "numericPrecisionRadix" : 10,
      "isCaseSensitive" : false,
      "isSearchable" : true,
      "isSigned" : true,
      "isCurrency" : false,
      "columnDisplaySize" : 1,
      "baseType" : null
    },
    "comment" : "",
    "collation" : "",
    "defaultDefinition" : ""
  }, {
    "name" : "DECIMAL_1_1_COL",
    "type" : {
      "name" : "DECIMAL",
      "typeNumber" : 3,
      "className" : "java.math.BigDecimal",
      "precision" : 1,
      "scale" : 1,
      "isNullable" : true,
      "isAutoIncrement" : false,
      "numericPrecisionRadix" : 10,
      "isCaseSensitive" : false,
      "isSearchable" : true,
      "isSigned" : true,
      "isCurrency" : false,
      "columnDisplaySize" : 1,
      "baseType" : null
    },
    "comment" : "",
    "collation" : "",
    "defaultDefinition" : ""
  }, {
    "name" : "NUMERIC_1_1_COL",
    "type" : {
      "name" : "NUMERIC",
      "typeNumber" : 2,
      "className" : "java.math.BigDecimal",
      "precision" : 1,
      "scale" : 1,
      "isNullable" : true,
      "isAutoIncrement" : false,
      "numericPrecisionRadix" : 10,
      "isCaseSensitive" : false,
      "isSearchable" : true,
      "isSigned" : true,
      "isCurrency" : false,
      "columnDisplaySize" : 1,
      "baseType" : null
    },
    "comment" : "",
    "collation" : "",
    "defaultDefinition" : ""
  }, {
    "name" : "DEC_100000_COL",
    "type" : {
      "name" : "DECIMAL",
      "typeNumber" : 3,
      "className" : "java.math.BigDecimal",
      "precision" : 100000,
      "scale" : 0,
      "isNullable" : true,
      "isAutoIncrement" : false,
      "numericPrecisionRadix" : 10,
      "isCaseSensitive" : false,
      "isSearchable" : true,
      "isSigned" : true,
      "isCurrency" : false,
      "columnDisplaySize" : 100000,
      "baseType" : null
    },
    "comment" : "",
    "collation" : "",
    "defaultDefinition" : ""
  }, {
    "name" : "DECIMAL_100000_COL",
    "type" : {
      "name" : "DECIMAL",
      "typeNumber" : 3,
      "className" : "java.math.BigDecimal",
      "precision" : 100000,
      "scale" : 0,
      "isNullable" : true,
      "isAutoIncrement" : false,
      "numericPrecisionRadix" : 10,
      "isCaseSensitive" : false,
      "isSearchable" : true,
      "isSigned" : true,
      "isCurrency" : false,
      "columnDisplaySize" : 100000,
      "baseType" : null
    },
    "comment" : "",
    "collation" : "",
    "defaultDefinition" : ""
  }, {
    "name" : "NUMERIC_100000_COL",
    "type" : {
      "name" : "NUMERIC",
      "typeNumber" : 2,
      "className" : "java.math.BigDecimal",
      "precision" : 100000,
      "scale" : 0,
      "isNullable" : true,
      "isAutoIncrement" : false,
      "numericPrecisionRadix" : 10,
      "isCaseSensitive" : false,
      "isSearchable" : true,
      "isSigned" : true,
      "isCurrency" : false,
      "columnDisplaySize" : 100000,
      "baseType" : null
    },
    "comment" : "",
    "collation" : "",
    "defaultDefinition" : ""
  }, {
    "name" : "DEC_100000_0_COL",
    "type" : {
      "name" : "DECIMAL",
      "typeNumber" : 3,
      "className" : "java.math.BigDecimal",
      "precision" : 100000,
      "scale" : 0,
      "isNullable" : true,
      "isAutoIncrement" : false,
      "numericPrecisionRadix" : 10,
      "isCaseSensitive" : false,
      "isSearchable" : true,
      "isSigned" : true,
      "isCurrency" : false,
      "columnDisplaySize" : 100000,
      "baseType" : null
    },
    "comment" : "",
    "collation" : "",
    "defaultDefinition" : ""
  }, {
    "name" : "DECIMAL_100000_0_COL",
    "type" : {
      "name" : "DECIMAL",
      "typeNumber" : 3,
      "className" : "java.math.BigDecimal",
      "precision" : 100000,
      "scale" : 0,
      "isNullable" : true,
      "isAutoIncrement" : false,
      "numericPrecisionRadix" : 10,
      "isCaseSensitive" : false,
      "isSearchable" : true,
      "isSigned" : true,
      "isCurrency" : false,
      "columnDisplaySize" : 100000,
      "baseType" : null
    },
    "comment" : "",
    "collation" : "",
    "defaultDefinition" : ""
  }, {
    "name" : "NUMERIC_100000_0_COL",
    "type" : {
      "name" : "NUMERIC",
      "typeNumber" : 2,
      "className" : "java.math.BigDecimal",
      "precision" : 100000,
      "scale" : 0,
      "isNullable" : true,
      "isAutoIncrement" : false,
      "numericPrecisionRadix" : 10,
      "isCaseSensitive" : false,
      "isSearchable" : true,
      "isSigned" : true,
      "isCurrency" : false,
      "columnDisplaySize" : 100000,
      "baseType" : null
    },
    "comment" : "",
    "collation" : "",
    "defaultDefinition" : ""
  }, {
    "name" : "DEC_100000_1_COL",
    "type" : {
      "name" : "DECIMAL",
      "typeNumber" : 3,
      "className" : "java.math.BigDecimal",
      "precision" : 100000,
      "scale" : 1,
      "isNullable" : true,
      "isAutoIncrement" : false,
      "numericPrecisionRadix" : 10,
      "isCaseSensitive" : false,
      "isSearchable" : true,
      "isSigned" : true,
      "isCurrency" : false,
      "columnDisplaySize" : 100000,
      "baseType" : null
    },
    "comment" : "",
    "collation" : "",
    "defaultDefinition" : ""
  }, {
    "name" : "DECIMAL_100000_1_COL",
    "type" : {
      "name" : "DECIMAL",
      "typeNumber" : 3,
      "className" : "java.math.BigDecimal",
      "precision" : 100000,
      "scale" : 1,
      "isNullable" : true,
      "isAutoIncrement" : false,
      "numericPrecisionRadix" : 10,
      "isCaseSensitive" : false,
      "isSearchable" : true,
      "isSigned" : true,
      "isCurrency" : false,
      "columnDisplaySize" : 100000,
      "baseType" : null
    },
    "comment" : "",
    "collation" : "",
    "defaultDefinition" : ""
  }, {
    "name" : "NUMERIC_100000_1_COL",
    "type" : {
      "name" : "NUMERIC",
      "typeNumber" : 2,
      "className" : "java.math.BigDecimal",
      "precision" : 100000,
      "scale" : 1,
      "isNullable" : true,
      "isAutoIncrement" : false,
      "numericPrecisionRadix" : 10,
      "isCaseSensitive" : false,
      "isSearchable" : true,
      "isSigned" : true,
      "isCurrency" : false,
      "columnDisplaySize" : 100000,
      "baseType" : null
    },
    "comment" : "",
    "collation" : "",
    "defaultDefinition" : ""
  }, {
    "name" : "DEC_100000_100000_COL",
    "type" : {
      "name" : "DECIMAL",
      "typeNumber" : 3,
      "className" : "java.math.BigDecimal",
      "precision" : 100000,
      "scale" : 100000,
      "isNullable" : true,
      "isAutoIncrement" : false,
      "numericPrecisionRadix" : 10,
      "isCaseSensitive" : false,
      "isSearchable" : true,
      "isSigned" : true,
      "isCurrency" : false,
      "columnDisplaySize" : 100000,
      "baseType" : null
    },
    "comment" : "",
    "collation" : "",
    "defaultDefinition" : ""
  }, {
    "name" : "DECIMAL_100000_100000_COL",
    "type" : {
      "name" : "DECIMAL",
      "typeNumber" : 3,
      "className" : "java.math.BigDecimal",
      "precision" : 100000,
      "scale" : 100000,
      "isNullable" : true,
      "isAutoIncrement" : false,
      "numericPrecisionRadix" : 10,
      "isCaseSensitive" : false,
      "isSearchable" : true,
      "isSigned" : true,
      "isCurrency" : false,
      "columnDisplaySize" : 100000,
      "baseType" : null
    },
    "comment" : "",
    "collation" : "",
    "defaultDefinition" : ""
  }, {
    "name" : "NUMERIC_100000_100000_COL",
    "type" : {
      "name" : "NUMERIC",
      "typeNumber" : 2,
      "className" : "java.math.BigDecimal",
      "precision" : 100000,
      "scale" : 100000,
      "isNullable" : true,
      "isAutoIncrement" : false,
      "numericPrecisionRadix" : 10,
      "isCaseSensitive" : false,
      "isSearchable" : true,
      "isSigned" : true,
      "isCurrency" : false,
      "columnDisplaySize" : 100000,
      "baseType" : null
    },
    "comment" : "",
    "collation" : "",
    "defaultDefinition" : ""
  } ],
  "indexes" : [ ],
  "collation" : "",
  "primaryKey" : null,
  "bestRowIdentifier" : null
}